cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -569,4 +582,22 @@ and then restart CDAP.
 #. Restart the CDAP processes::
 
      for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i start ; done
+     
+If you are upgrading a secure Hadoop cluster, see the notes above about
+:ref:`configuring secure Hadoop <install-secure-hadoop>`. 
+
+.. _install-troubleshooting:
+
+Troubleshooting
+---------------------------------
+
+- If you have YARN configured to use LinuxContainerExecutor (see the setting for
+  ``yarn.nodemanager.container-executor.class``), the ``cdap`` user needs to be present on
+  all Hadoop nodes.
+
+- If you are using a LinuxContainerExecutor, and the UID for the ``cdap`` user is less than
+  500, you will need to add the ``cdap`` user to the allowed users configuration for the
+  LinuxContainerExecutor in Yarn by editing the ``/etc/hadoop/conf/container-executor.cfg``
+  file. Change the line for ``allowed.system.users`` to::
 
+    allowed.system.users=yarn,cdap
```

Comment:
Actually make this just:

allowed.system.users=cdap

"yarn" is not normally listed there.  That is just another artifact of our old coopr templates running secure clusters as yarn.